README.md

```diff
@@ -42,57 +42,83 @@ Unlike other alternatives, such as fitnessyncer.com, this aims to offer very fin
 ----------------------------
 You have to register your own Fitbit and Google Fit applications. This setup is a one time thing.
 
-1. Install dependencies
+1. Run App.py
 -------------------
 This is a python3 application so install all the dependencies 
 
-- Create virtualenv ```virtualenv fitbitenv```
-- Activate env ```source fitbitenv/bin/activate``` 
-- Install dependencies using ```pip3 install -r requirements.txt```
+- Make sure you have Python 3.5 or higher installed ```sudo apt update && sudo apt install python3```
```

Comment:
Could we avoid platform specific commands or give an alternative for each platform? Maybe a link for Windows case.

Reply:
When I get back home Sunday, I'll pull out my old Windows laptop and see if I can get Windows 10 installed on it. From reading https://www.howtogeek.com/265900/everything-you-can-do-with-windows-10s-new-bash-shell/ it seems that Windows might be able to run bash commands, including apt-get, without issue. If it does throw any errors on Windows, I'll have it detect the underlying OS and run a separate set of commands based on Windows vs Linux.

Reply:
That would be great! Thank you.